src/components/Inputs/ErrorSpan.tsx

```diff
@@ -0,0 +1,23 @@
+import { faCircleExclamation } from '@fortawesome/free-solid-svg-icons';
```

Comment:
can we rename this to be more descriptive than 'errorSpan'

Reply:
yeha